shell/components/Resource/Detail/Card/StateCard/composables.ts

```diff
@@ -0,0 +1,111 @@
+import { extractCounts, Props as ResourceRowProps } from '@shell/components/Resource/Detail/ResourceRow.vue';
+import { useI18n } from '@shell/composables/useI18n';
+import { INGRESS, SERVICE } from '@shell/config/types';
+import { getHighestAlertColor } from '@shell/utils/style';
+import { computed, Ref, toValue } from 'vue';
+import { useStore } from 'vuex';
+import { Props as StateCardProps } from '@shell/components/Resource/Detail/Card/StateCard/index.vue';
+
+export function usesResourceCardRow(label: string, resources: any[], to?: any): ResourceRowProps {
```

Comment:
The `uses` naming is inconsistent with the rest of the composables used throughout this file

```suggestion
export function useResourceCardRow(label: string, resources: any[], to?: any): ResourceRowProps {
```

Reply:
Fixed. I did it on my end so I could do all the changes locally.